config/certmanager/certificate.yaml

```diff
@@ -1,25 +0,0 @@
-# The following manifests contain a self-signed issuer CR and a certificate CR.
-# More document can be found at https://docs.cert-manager.io
-# WARNING: Targets CertManager v1.0. Check https://cert-manager.io/docs/installation/upgrading/ for breaking changes.
-apiVersion: cert-manager.io/v1
-kind: Issuer
-metadata:
```

Comment:
why?! removed?

Reply:
It was not used because the `kustomization.yaml` in `config/default` had it commented out. I figure we should remove it since it's just clutter.